packages/resolve-url/resolve-url.js

```diff
@@ -0,0 +1,20 @@
+import isAbssoluteUrl from 'is-absolute-url';
+
+const resolveUrl = url => {
+  if (isAbssoluteUrl(url)) {
+    return url;
+  }
+  const { href, pathname } = window.location;
```

Comment:
This doesn't account for the [base tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base)

Reply:
@TheSharpieOne are you suggesting:

```js
const { href } = document.querySelector('base');
```